main.py

```diff
@@ -1,16 +1,27 @@
-# This is a sample Python script.
+from selenium import webdriver, common
+from selenium.webdriver.chrome.service import Service
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
 
-# Press ⌃R to execute it or replace it with your code.
-# Press Double ⇧ to search everywhere for classes, files, tool windows, actions, and settings.
+# Немного настроим наш браузер
+driver = webdriver.Chrome(service=Service('/Users/yuliametel/PycharmProjects/OwnTestFramework/venv/lib/python3.9'
```

Comment:
Please, research, how to specify path without hardcode yuliametel. It's a bad way, path should be calculated dynamically

Reply:
Now there is problem with automatically finding of the driver by path. The driver is located in the right path, but the error does not disappear. It will fix in next pull request.